.vscode/settings.json

```diff
@@ -16,6 +16,17 @@
   },
   "protoc": {
     "compile_on_save": true,
-    "options": ["--proto_path=protos", "--dart_out=mobile/lib/protos", "--python_out=backend/protos", "--js_out=web/src/protos"]
-  }
+    "options": [
+      "--dart_out=grpc:mobile/lib",
+      "--python_out=backend",
+      "--grpc_python_out=backend",
+      "--plugin=protoc-gen-grpc_python=/usr/bin/grpc_python_plugin",
```

Comment:
Est ce que l'on pourrait plutôt placer `grpc_python_plugin` dans `/usr/local/bin` plutôt que dans `/usr/bin`? (si oui, il faudrait aussi changer https://github.com/PolyCortex/polydodo/pull/24/files#diff-67309dda209555c658e1a0c73ef33406R4)
La première raison est que, sur mac, tu ne peux plus ajouter, même en sudo, à `/usr/bin`. :'( 
La deuxième raison est que, selon ce stackoverflow, "future distribution upgrades may modify or delete them without warning". Donc, selon ça, c'est conseillé de le mettre dans `/usr/local/bin`(https://unix.stackexchange.com/questions/8656/usr-bin-vs-usr-local-bin-on-linux)

Reply:
J'ai ajouté une section par rapport à grpc dans le readme: https://github.com/PolyCortex/polydodo/blob/master/README.md#grpc

Dans le fond, ubuntu, alpine et arch linux ont un package qui s'appelle grpc et ce package inclut le plugin python, donc pas besoin de faire l'installation manuellement avec le script.

Ceci étant dit, `/usr/bin` n'est en effet pas cross-platform, il faut trouver une meilleure solution

Reply:
Une solution serait d'enlever cette ligne et de rouler:

`ln -nsf $(which grpc_python_plugin) /some/where/in/your/path/protoc-gen-grpc_python`

Reply:
Pour une raison qui m'échappe, je dois spécifier `"--plugin=$(which protoc-gen-grpc_python)"` plutôt que d'écrire directement `--plugin=protoc-gen-grpc_python`, sans quoi j'obtiens un erreur qui me dit qu'il n'est pas dans le path. (alors qu'il est bien dans mon path?). 
Est ce que t'as une idée pourquoi ça ne marche pas sans le which? Sinon, selon moi, je pense que c'est quand même correct de mettre avec le which, comme ça devrait marcher dans tous les cas.

Reply:
Peut-être que le script qui l'ajoute à ton path n'est pas utilisé par l'extension qui compile les protos?

Reply:
Si tu fais le symlink, y a pas besoin d'écrire --plugin.

D'ailleurs, peut-on l'enlever de vscode pls?